examples/files.json

```diff
@@ -0,0 +1,44 @@
+{
+    "files": [
+        {
+            "_id": "a34e4793-c343-4930-973f-a45196289017",
+            "id": "a34e4793-c343-4930-973f-a45196289017",
+            "name": "dataset1_file1.fastq",
+            "format": "FASTQ",
+            "type": "Unaligned Reads",
+            "size": "432432423",
+            "md5sum": "9aa9aabaaf67287c332d85c0de825c03",
+            "category": "Whole Genome Sequencing"
+        },
+        {
+            "_id": "9e9d1ec1-aadc-4f45-a8e8-90d1e66f827f",
```

Comment:
Why an `_id` as well as `id`?

Reply:
The `_id` is internal in MongoDB. We want to keep the internal ID the same as our GHGA assigned ID for entities. 

Keeping the internal ID same as the entity ID allows for defining manual references between documents in different collections.

Reply:
Why can't we just use one field for both? Just `_id` I mean?

Reply:
Because I'd like the `id` field to be exposed to users whereas `_id` can be internal and technology specific. This is still TBD since we are mulling things over on what db technology we want to be using. 

For all intends and purposes - `_id` should never be used for querying. Its an internal representation that shadows `id` field at the moment (and can change in the future)

Reply:
I am not sure I'm fully on board with this, since it's duplicating fields/codes in the one case, and perhaps unnecessarily implementing two id codes in the other, though I can see why we would want a more private id. Just my two cents though.

Reply:
Yes, I agree with you regarding duplicating fields that holds the same information. The schema (https://github.com/ghga-de/sandbox-metadata/blob/feature/GDEV-48/metadata_service/models.py#L36) doesn't specify two IDs for the File object. Its just there for the sake of bootstrapping the application (and temporary). Subsequent PRs will get rid of the `_id`.

Reply:
Great! 😁

Reply:
We could allow MongoDB to generate its own _id field instead of using the id value for _id field.